lib/omnibus/health_check.rb

```diff
@@ -51,6 +51,17 @@ class HealthCheck
       /libutil\.so/,
     ].freeze
 
+    WRLINUX_WHITELIST_LIBS = [
+      /libc\.so/,
+      /libcrypt\.so/,
+      /libdl\.so/,
+      /libffi\.so/,
```

Comment:
We compile `libffi`. Excluding it from the health check is dangerous. Everything else already exists in the default `WHITELIST_LIBS `.